tests/Unit/ZenstruckFoundryBundleTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the zenstruck/foundry package.
+ *
+ * (c) Kevin Bond <kevinbond@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zenstruck\Foundry\Tests\Unit;
+
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\Configurator\DefinitionConfigurator;
+use Symfony\Component\Config\Definition\Loader\DefinitionFileLoader;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\Config\FileLocator;
+use Zenstruck\Foundry\ORM\ResetDatabase\ResetDatabaseMode;
+use Zenstruck\Foundry\ZenstruckFoundryBundle;
+
+final class ZenstruckFoundryBundleTest extends TestCase
```

Comment:
I'm not against to at this test to our test suite, but I'm not sure it could assert the bug you fixed won't happen again :thinking: It does not use a container, nor use the loads the extension

I think we should do an integration test for this, see my comment here https://github.com/zenstruck/foundry/pull/877#issuecomment-2804119865

Reply:
ok! :ok_hand:

Reply:
BTW, don't hesitate to do all those tests in the same PR :)

Reply:
Ahhh okay sure!